services/movie_session.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+    return movie_session
+
+
+def get_movies_sessions(
+        session_date: Optional[datetime] = None
+) -> QuerySet[MovieSession]:
+    movie_session = MovieSession.objects.all()
+    if session_date:
+        movie_session = MovieSession.objects.filter(
+            show_time__date=session_date
+        )
+
+    return movie_session
+
+
+def get_movie_session_by_id(
+        movie_session_id: int
+) -> MovieSession:
+
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(session_id: int,
+                         show_time: Optional[datetime] = None,
+                         movie_id: Optional[int] = None,
+                         cinema_hall_id: Optional[int] = None) -> None:
+    if show_time:
+        MovieSession.objects.filter(id=session_id).update(
+            show_time=show_time
+        )
+    if movie_id:
+        MovieSession.objects.filter(id=session_id).update(
+            movie_id=movie_id
+        )
+    if cinema_hall_id:
+        MovieSession.objects.filter(id=session_id).update(
+            cinema_hall_id=cinema_hall_id
+        )
```

Comment:
Triple update is not good. Simplify code. Use or two queries(get + save) or one query(forming update params and that do .update)